README.md

```diff
@@ -1,14 +1,15 @@
-# HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
+# HarbourBridge: Turnkey Postgres-to-Spanner/MySQL-to-Spanner Evaluation
+
 [![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
 
 HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
-using data from an existing PostgreSQL database. The tool ingests pg_dump
-output, automatically builds a Spanner schema, and creates a new Spanner
-database populated with data from pg_dump.
+using data from an existing PostgreSQL/MySQL database. The tool ingests from
+database/dump file output, automatically builds a Spanner schema, and creates
```

Comment:
How about:

The tool ingests schema and data from either a pgdump/mysqldump file or directly from the database,
automatically builds ...